### DISCUSSION TITLE
Error: channel does not exist <in this version of the universe>

### DISCUSSION THREAD

**Discussion by ghost** [Tyrrrz/DiscordChatExporter#711] [Q&A] [answered]
"ERROR
DiscordChatExporter.Core.Exceptions.DiscordChatExporterException: Requested resource (channels/1234) does not exist..."

Where 1234 is the channel ID copied from Discord.

I'm out of ideas for what to try next.

Does anyone else have any?

Thank you for your time,
Marie

[Screen Shot .pdf](https://github.com/Tyrrrz/DiscordChatExporter/files/7221238/Screen.Shot.pdf)

**Comment by 96-LB** [comment, marked as answer]
Are you trying to export a DM channel?

**Reply by ghost** [reply]
Yes. : ) Sorry context would be helpful... I am trying to export for the very first time on any channel ever. I tried with two different channels and get the same error. I use Discord mostly for teaching and this for giving extra participation credit. Scrolling through is uhm, burdensome. So I hoped this would help.

**Reply by 96-LB** [reply]
Check out [this wiki page](https://github.com/Tyrrrz/DiscordChatExporter/wiki/Obtaining-Token-and-Channel-IDs#how-to-get-a-direct-message-channel-id) for directions. DM channels are a little bit different because hitting "Copy ID" gives you the user's ID instead of the channel's.

**Reply by ghost** [reply]
THANK YOU!!!!
